lib/index.js

```diff
@@ -267,11 +274,21 @@ export default {
             range = Helpers.generateRange(textEditor, line);
           }
 
+          // avoid duplicated issues
+          if (fingerprints.has(issue.fingerprint)) {
+            return;
+          }
+          fingerprints.add(issue.fingerprint);
+
           linterResults.push({
-            type: 'Warning',
-            text: issue.description,
-            filePath,
-            range,
+            severity: mapSeverity[issue.severity] || 'warning',
+            excerpt: `${issue.engine_name.toUpperCase()} [${issue.check_name}]: ${issue.description}`,
+            description: (issue.content && issue.content.body) ? issue.content.body : undefined,
+            url: `${issue.engine_name} ${issue.check_name}`,
```

Comment:
This needs to be a real URL, as in `https://domain.tld/path/to/page`.

(The search functionality I'm guessing you were taking advantage of here was removed in v1.2.3 of `linter-ui-default`)